### PR TITLE
Refs [TASK][26666] Sync user block status with actions

### PR DESCRIPTION
**Pull request by hoatb-1022**
https://edu-redmine.sun-asterisk.vn/issues/26666

Screen Shot 2020-08-04 at 14 59 32

### CODE REVIEW

**Comment by hoatb-1022** [comment]
ready

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+      flash[:error] = t ".blocked_account"
+      redirect_to login_path
+    else
+      super
+    end
+  end
+
+  private
+
+  def find_user_by_email
```
Comment (ngocvt-0484) [inline app/controllers/users/sessions_controller.rb]: load_user dc hơn e

File: app/controllers/users/sessions_controller.rb [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+    @user = User.find_by email: params[:user][:email].downcase
```
Comment (ngocvt-0484) [inline app/controllers/users/sessions_controller.rb:18]: xử lý trường hợp k tìm thấy

**Comment by hoatb-1022** [comment]
ready

**Comment by hoatb-1022** [comment]
ready

**Pull request merged**